cmd/kubelet/kubelet.go

```diff
@@ -57,7 +57,7 @@ var (
 	registryBurst           = flag.Int("registry_burst", 10, "Maximum size of a bursty pulls, temporarily allows pulls to burst to this number, while still not exceeding registry_qps.  Only used if --registry_qps > 0")
 	runonce                 = flag.Bool("runonce", false, "If true, exit after spawning pods from local manifests or remote urls. Exclusive with --etcd_servers and --enable-server")
 	enableDebuggingHandlers = flag.Bool("enable_debugging_handlers", true, "Enables server endpoints for log collection and local running of containers and commands")
-	minimumGCAge            = flag.Duration("minimum_container_ttl_duration", 0, "Minimum age for a finished container before it is garbage collected.  Examples: '300ms', '10s' or '2h45m'")
+	minimumGCAge            = flag.Duration("minimum_container_ttl_duration", 10*time.Second, "Minimum age for a finished container before it is garbage collected.  Examples: '300ms', '10s' or '2h45m'")
```

Comment:
Hmm, given that the default was 0 which mean "instantly" I guess I could live with 1 minute here